### PR TITLE
Updating worker status when TaskTimedOutException is caught.

### PR DESCRIPTION
**Pull request by anderjo4**

### CODE REVIEW

File: cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java [hunk 1]
```diff
@@ -181,7 +181,9 @@ public void run() {
 							logger.warn("CommandRuntimeException: " + e.getErrorName(), e);
 							sendTaskResultBestEffort(taskResult, e);
 						} catch (TaskTimedOutException e) {
-							logger.error("AWS Task has timed out while processed by the Worker (heartbeats too sparse or total time exceeded).", e);
+							String errorMessage = "AWS Task has timed out while processed by the Worker (heartbeats too sparse or total time exceeded).";
+							updateStatusReport("", new Date(), errorMessage);
```
Comment (takesson) [inline cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java]: "Timeout" in first param?

File: cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java [hunk 2]
```diff
@@ -385,7 +387,9 @@ private void sendTaskResultBestEffort(GetActivityTaskResult taskResult, CommandR
 		try {
 			sendTaskResult(taskResult, cre);
 		} catch (TaskTimedOutException timedOutEx) {
-			logger.error("AWS Task has timed out while processed by the Worker (heartbeats too sparse or total time exceeded).", timedOutEx);
+			String errorMessage = "AWS Task has timed out while processed by the Worker (heartbeats too sparse or total time exceeded).";
+			updateStatusReport("", new Date(), errorMessage);
```
Comment (takesson) [inline cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java]: "Timeout" in first param?

File: cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java (hunk 1 above)
Comment (takesson) [reply cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java]: "Task Timeout" in first param?

File: cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java (hunk 2 above)
Comment (takesson) [reply cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java]: "Task Timeout" in first param?

Comment (anderjo4) [reply cms-publish-worker-webapp/src/main/java/se/simonsoft/cms/publish/worker/AwsStepfunctionPublishWorker.java]: ok

**Pull request merged**